locales/en/messages.json

```diff
@@ -2579,6 +2579,43 @@
         "message": "Start over"
     },
 
+    "escDshotDirectionDialogTitle": {
+        "message": "Motor direction"
+    },
+    "escDshotDirectionSelectMotor": {
+        "message": "Select a motor"
+    },
+    "escDshotDirectionRiskNotice": {
+        "message": "<strong>Safety notice</strong><br /><strong class=\"message-negative\">Remove all propellers to prevent injury!</strong><br />The motors will <strong>spin up!</strong>"
+    },
+    "escDshotDirectionDialogUnderstandRisks": {
+        "message": "<strong>I understand the risks</strong>,<br />the propellers are removed."
+    },
+    "escDshotDirectionDialogExplanations": {
+        "message": "<strong>Information notice</strong><br />A motors will spin up once you select it and push \"Test\". The battery should be plugged in, correct ESC protocol should be selected."
+    },
+    "dialogEscDshotDirectionClose": {
+        "message": "Close"
+    },
+    "dialogEscDshotDirectionOpen": {
+        "message": "Fix motor direction"
```

Comment:
Good job Lemon lets try it, I think it should be named "Motor direction" like dialog tittle.